### PR TITLE
sort envs in app info

### LINKED ISSUE #336: Sort env vars.
It would be nice if `teresa app info myapp` return env vars sorted.

### PR DESCRIPTION
**Pull request by yagonobre**
fixes #336

---
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa/337)

### CODE REVIEW

File: pkg/server/app/sort.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package app
```
Comment (drgarcia1986) [inline pkg/server/app/sort.go]: why not in `pkg/server/client` ?

File: pkg/server/app/sort.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import "sort"
+
+type ByKey []*EnvVar
```
Comment (drgarcia1986) [inline pkg/server/app/sort.go]: maybe `EnvVarsByKey`?

Comment (yagonobre) [reply pkg/server/app/sort.go]: I think that `ByKey` is more cleaner, like in sort documentation

Comment (drgarcia1986) [reply pkg/server/app/sort.go]: but think about usave 
```go
import "pkg/client/app/sort"
...
sort.ByKey
```
`ByKey` of what?

Comment (yagonobre) [reply pkg/server/app/sort.go]: you can use the func `sortEnvsByKey`

**Pull request merged**